skills/vendure-cli/commands/start.md

```
@@ -0,0 +1,44 @@
+# vendure start
+
+Runs a project that has already been compiled with `vendure build`.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)